lib/job.js

```diff
@@ -53,6 +53,10 @@ function setDefaultOpts(opts) {
   _opts.attempts = parseInt(_opts.attempts);
   _opts.backoff = backoffs.normalize(_opts.backoff);
 
+  if (_opts.sizeLimit) {
```

Comment:
I think it is better to keep this option as a Queue option, so that all jobs added to that particular instance of a queue will need to honor the size.

Reply:
Sounds good, I didn't realize that was how it works, I'll make the change.